File: 이코테/P96_숫자카드게임/P96.c
#include<stdio.h>

int main()
{
    int m,n;
    scanf("%d %d",&m,&n);

    int result = 0;
    for(int i=0 ; i<m ; i++)
    {
        int candidate = 10001;
        for(int j=0 ; j<n ; j++)
        {
            int temp;
            scanf("%d", &temp);
            if(temp < candidate) candidate = temp;
        }

        if(result < candidate) result = candidate;
    }

    printf("%d", result);
}

File: 이코테/P96_숫자카드게임/P96.py
n,m = map(int, input().split())

result = 0
for i in range(n):
    candidate = 10001
    for j in list(map(int, input().split())):
        if j < candidate:
            candidate = j
    if(result < candidate):
        result = candidate

print(result)
